src/test/java/io/jenkins/plugins/audit/listeners/ItemChangeListenerTest.java

```diff
@@ -48,7 +47,7 @@ public void testAuditOnItemCreate() throws Exception {
         FreeStyleProject project = j.createFreeStyleProject("Test build");
 //        project.getBuildersList().add(new Shell("echo Test audit-log-plugin"));
 
-        assertEquals("Events on item creation not triggered", 1, events.size());
+        assertThat(((AuditMessage)events.get(0).getMessage()).getId().toString()).isEqualTo("createItem");
```

Comment:
So I tried out assertJ a bit myself to see what methods they have that might be useful. Here are some examples of using it that seem to work, though they're not all perfect. It's a good starting point at least for your own reference:

```
        assertThat(events).hasSize(1);
        assertThat(events).extracting(LogEvent::getMessage).allMatch(AuditMessage.class::isInstance);
        assertThat(events).extracting(event -> ((AuditMessage) event.getMessage()).getId().toString()).allMatch(id -> id.equals("createItem"));
```

For the later tests, you can use the extracting from the last line here but instead use one of the `containsFoo` assertions instead of `allMatch`. This gives you a more fluent way to express your test assertion without a lot of unnecessary code.